init.lua

```diff
@@ -1,77 +1,302 @@
+local mod_name = "wielded_light"
+
 local update_interval = 0.2
+local cleanup_interval = update_interval*3
+local velocity_projection = update_interval * 1
 local level_delta = 2
+local max_light_level = 14
```

Comment:
Use `minetest.LIGHT_MAX`

Reply:
That's done, it will change behaviour on any server where this is not 14 though - as this was previously hard coded as 14